webapp/src/router/index.ts

```diff
@@ -0,0 +1,23 @@
+// Composables
+import { createRouter, createWebHistory } from 'vue-router';
+
+const routes = [{
+  path: '/',
+  component: () => import('@/layouts/default/Default.vue'),
+  children: [{
+    path: '',
+    name: 'Home',
+    // route level code-splitting
+    // this generates a separate chunk (about.[hash].js) for this route
+    // which is lazy-loaded when the route is visited.
+    component: () =>
+      import(/* webpackChunkName: "home" */ '@/views/Home.vue'),
+  },],
+},];
```

Comment:
We will use a Multi Page Application structure during the transition, so I think we can remove the router for this initial setup.

It is unsure if we will be using the router for smaller routing feature.

Reply:
To embed the new pages/views inside the legacy app, we will use the MPA architecture (multi page application), so the routing will be done server side.

It will only once we have all the views put together and that we can drop the essential block of legacy that we will be able to use the SPA architecture.

So unless I am missing something, I don't understand how this will be easier.

Reply:
It's substantially easier to use the router than custom-coding a MPA solution. In the research I've been doing, it's been recommended to create multiple SPAs (with only one view) and compile them all separately into multiple page apps if you really need an MPA: https://github.com/wandersonpjbkn/vue-mpa-application-example. Doing _n_ setups (where _n_ is the number of pages) seems excessive.